src/metabase/lib/convert.cljc

```diff
@@ -175,7 +175,7 @@
                           (mapv (fn [[k v]]
                                   (-> v
                                       ->pMBQL
-                                      (lib.util/named-expression-clause k))))
+                                      (lib.util/top-level-expression-clause k))))
```

Comment:
Unfortunately this PR breaks `Lib.withExpressionName` - it has no effect on custom columns.
It works in `master`.
It causes [FE unit tests to fail](https://github.com/metabase/metabase/actions/runs/7140501215/job/19445925676?pr=36455#step:6:1701) in #36455.